pipeline/run_beam_tables.py

```diff
@@ -112,7 +112,8 @@ def run_user_pipelines(runner: beam_tables.ScanDataBeamPipelineRunner,
       end_date=end_day)
 
 
-def get_firehook_beam_pipeline_runner():
+def get_firehook_beam_pipeline_runner(
+) -> beam_tables.ScanDataBeamPipelineRunner:
```

Comment:
No action needed, just... that formatting is gross.

Reply:
haha agreed